pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -113,14 +120,14 @@ func (p *Provider) EnsureAll(ctx context.Context, nodeClass *v1beta1.EC2NodeClas
 	if err != nil {
 		return nil, err
 	}
-	launchTemplates := map[string][]*cloudprovider.InstanceType{}
+	launchTemplates := []*LaunchTemplate{}
```

Comment:
```suggestion
	var launchTemplates []*LaunchTemplate{}
```

Prefer this syntax: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices